### PR TITLE
Display group's image if there is one

### PR DESCRIPTION
**Pull request by JacobBaqleh1**
Issue: #<Number>

## Summary (1-2 sentences)

On the groups.tsx page, individual groups now display their group image. If they do not have a group image, they have the default photo displayed.

## Details (reason and description of the changes)

Before, every group had the default group image displayed. Now that there is group image uploading, I added the groups image from the db into the image element as a ternary operator.

### Alternative implementations (what alternatives were considered and why were they disregarded?) (optional)

I made the background image an empty string because group images that have transparent backgrounds would have the background image behind them. I am not sure if this was the best approach to handle transparent images because I think the background image is important to have as a skeleton. 

## Screenshots/Recordings (if applicable)
Before:
image
After:
image

Here is with and without the background image.
Before:
image

After:
image

### CODE REVIEW

File: app/routes/groups.tsx [hunk 1]
```diff
@@ -69,9 +69,17 @@ export default function Group() {
                     <div className="w-52 h-52">
                       <Image
                         className="bg-contain bg-center flex self-start rounded-full"
-                        src="https://images.pexels.com/photos/3184339/pexels-photo-3184339.jpeg"
+                        src={
+                          group.imgUrl
+                            ? group.imgUrl
+                            : 'https://images.pexels.com/photos/3184339/pexels-photo-3184339.jpeg'
```
Comment (eiffelwong1) [inline app/routes/groups.tsx:75]:
```suggestion
                          group.imgUrl ?? 'https://images.pexels.com/photos/3184339/pexels-photo-3184339.jpeg'
```

File: app/routes/groups.tsx [hunk 2]
```diff
@@ -69,9 +69,17 @@ export default function Group() {
...
+                        }
                         alt="Group meeting around an office table"
-                        background="https://res.cloudinary.com/dxctpvd8v/image/upload/v1709096811/SocialPlanit-Loading_qedebk.png"
+                        background={
+                          group.imgUrl
+                            ? ''
+                            : 'https://res.cloudinary.com/dxctpvd8v/image/upload/v1709096811/SocialPlanit-Loading_qedebk.png'
```
Comment (eiffelwong1) [inline app/routes/groups.tsx:81]:
```suggestion
                          group.imgUrl ?? 'https://res.cloudinary.com/dxctpvd8v/image/upload/v1709096811/SocialPlanit-Loading_qedebk.png'
```

**Review by eiffelwong1 — COMMENTED** [review]
have some code suggestions, overall looks good to me

File: app/routes/groups.tsx [hunk 3]
```diff
@@ -68,9 +68,12 @@ export default function Group() {
                     <div className="w-52 h-52">
                       <Image
                         className="bg-contain bg-center flex self-start rounded-full"
-                        src="https://images.pexels.com/photos/3184339/pexels-photo-3184339.jpeg"
+                        src={group.imgUrl ?? 'https://images.pexels.com/photos/3184339/pexels-photo-3184339.jpeg'}
                         alt="Group meeting around an office table"
-                        background="https://res.cloudinary.com/dxctpvd8v/image/upload/v1709096811/SocialPlanit-Loading_qedebk.png"
+                        background={
+                          group.imgUrl ??
```
[the remark this answers is not in the document]
Comment (CamilingJS) [reply app/routes/groups.tsx:74]: I would agree. 

low res img while loading:  "https://res.cloudinary.com/dxctpvd8v/image/upload/v1709096811/SocialPlanit-Loading_qedebk.png" 

img if no group.imgUrl: "https://images.pexels.com/photos/3184339/pexels-photo-3184339.jpeg"

**Review by CamilingJS — APPROVED** [review]
LGTM!

**Review by andrelandgraf — APPROVED** [review]
Thanks for making the changes! Ship it!

**Pull request merged**